apps/src/templates/sectionAssessments/StudentsAssessmentsMCTable.jsx

```diff
@@ -99,9 +98,10 @@ class StudentsAssessmentsMCTable extends Component {
   };
 
   submissionTimestampColumnFormatter = (submissionTimeStamp, {rowData}) => {
-    const submissionStatus = rowData.submissionStatus;
+    const isSubmitted = rowData.submissionStatus;
+    const submissionStatus = 'In progress';
```

Comment:
This constant is a bit confusing. Instead remove this constant, and on line 116, just return a localized string for 'In progress'.

```
} else {
  return i18n.inProgress();
}
```